modules/rosa-sts-creating-cluster.adoc

```diff
@@ -41,11 +60,7 @@ $ rosa create cluster \
   --support-role-arn arn:aws:iam::${aws_account_id}:role/ManagedOpenShift-Support-Role \
```

Comment:
The role-arn created by account-roles should be `--role-arn arn:aws:iam::301721915996:role/ManagedOpenShift-Installer-Role` not `--role-arn arn:aws:iam::${aws_account_id}:role/ManagedOpenShift-IAM-Role`

Reply:
Thank you for highlighting this. I have updated the example in the PR.